qplist.pro

```diff
@@ -0,0 +1,14 @@
+QT += xml
```

Comment:
These are some good points.  I think in the context of qt-pods naming it plist (or rwplist or whatever if there are naming conflicts) makes sense.  In the context of the rest of the world qtplist is probably a better name, because it conveys its Qt-ness, but that's not necessary in qt-pods, where everything has Qt-ness.